### DISCUSSION TITLE
[Feature Request] Hide options underneath Channel and Subscribe when you click on a video

### DISCUSSION THREAD

**Discussion by curiousdreamer** [qnblackcat/uYouPlus#1185] [Ideas]
So the options underneath the Youtube Channel, subscriibers, and the Subscribe button are the likes, dislikes, share, remix, thanks, download, clip, and share. My feature request would be able to hide these options if you want to. Mainly I just want to hide remix, clip, and thanks options.

**Comment by evkaw** [comment]
any update on this?
